client/src/components/App.jsx

```diff
@@ -94,11 +88,11 @@ const App = () => {
       if (i < fullStars) {
         stars.push(fullStar(i, 'f', key));
       } else if ( i > fullStars - 1) {
-        stars.push(fullStar(i, 'e', key))
+        stars.push(fullStar(i, 'e', key));
       }
     }
     return stars;
-  }
+  };
 
   return (
     <>
```

Comment:
This all looks good. We might want to consider moving these functions to their own file to make App.jsx look cleaner and be easier to read